homeassistant/components/auth/strings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "mfa_setup":{
+    "totp": {
+      "title": "TOTP",
+      "step": {
+        "init": {
+          "title": "Scan this QR code with your app",
+          "description": "Scan the QR code with your authentication app, such as **Google Authenticator** or **Authy**. \nOr enter {code} instead. \n\n[QR Code]{url}",
```

Comment:
Markdown image syntax is `![alt text](url)`. I don't see that in the description?